histomicstk/ComputeHaralickFeatures.py

```diff
@@ -0,0 +1,199 @@
+import numpy as np
+import pandas as pd
+import skimage.feature
+from skimage.measure import regionprops
+
+
+def ComputeHaralickFeatures(I, Label, Offsets=[[0, 1]], NumLevels=256,
+                            MaxGray=255, MinGray=0):
+    """
+    Calculates 26 Haralick features from an intensity image with the labels.
+
+    Parameters
+    ----------
+    I : array_like
+        M x N intensity image.
+    Label : array_like
+        M x N label image.
+    Offsets : array_like
+        Specifies common angles for a 2D image, given the pixel distacne D.
+        Default value = [[0, 1]]
+        AngleXY     OFFSET
+        -------     ------
+        0           [0 D]
+        45          [-D D]
+        90          [-D 0]
+        135         [-D -D]
+    NumLevels : integer
+        Specifies the number of gray level. NumLevels determines the size of
+        the gray-level co-occurrence matrix.
+        Default value = 256.
+    MaxGray : integer
+        Specifies a maximum grayscale value in I. Defalut value = 255
+    MinGray : integer
+        Specifies a minimum grayscale value in I. Defalut value = 0
```

Comment:
@slee172 I would suggest clubbing MaxGray and MinGray into a single argument like GrayLimits or IntensityRange.